app/views/govuk_publishing_components/components/_attachment.html.erb

```diff
@@ -46,20 +46,31 @@
     <% end %>
   <% end %>
 
-  <%= tag.h2 class: "gem-c-attachment__title" do %>
-    <%= link_to attachment.title, attachment.url,
-          class: "govuk-link",
-          target: target %>
-  <% end %>
+  <%= tag.div class: "gem-c-attachment__details" do %>
+    <%= tag.h2 class: "gem-c-attachment__title" do %>
+      <%= link_to attachment.title, attachment.url,
+            class: "govuk-link",
+            target: target %>
+    <% end %>
 
-  <% if attributes.any? %>
-    <%= tag.p raw(attributes.join(', ')), class: "gem-c-attachment__metadata" %>
-  <% end %>
+    <% if attributes.any? %>
+      <%= tag.p raw(attributes.join(', ')), class: "gem-c-attachment__metadata" %>
+    <% end %>
+
+    <% unless hide_help_text %>
```

Comment:
How about 'hide_opendocument_metadata'?

Reply:
Updated with the suggested name and rebased.